README.md

```diff
@@ -15,6 +15,9 @@ Synopsis
     # Build PostgreSQL server
     pgenv build 10.4
     
+    # Build from git
+    pgenv build-git
```

Comment:
I think we should refer to it as building current development, or `build-devel` or `build-dev`.

Reply:
My initial idea is to make it explicit that this will interact with `git`, and in fact this could build from any git repository you need. I've changed to `build-dev`, but as second thought I would retain the `build-git` or maybe `build-from-git`. The `dev` bit is not the important part to me. You could want to build `REL_11_STABLE` from git...